benchmarks/diffusion/main.py

```diff
@@ -19,7 +19,7 @@
 
 @dataclass
 class Arguments:
-    model: str = "runwayml/stable-diffusion-v1-5"
+    model: str = "benjamin-paine/stable-diffusion-v1-5"
```

Comment:
`HF_TOKEN` owner must agree the license on https://huggingface.co/benjamin-paine/stable-diffusion-v1-5 to gain access